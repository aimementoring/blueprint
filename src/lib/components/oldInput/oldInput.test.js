import React from 'react';
import renderer from 'react-test-renderer';
import OldInput from './oldInput';

describe('oldInput', () => {
  it('renders properly', () => {
    const tree = renderer
      .create(<OldInput placeholder="Name" value="" name="name" type="text" required disabled />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
