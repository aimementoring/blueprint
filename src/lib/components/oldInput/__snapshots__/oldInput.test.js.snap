// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`oldInput renders properly 1`] = `
<div
  className=" theme-purple inputWrapper"
>
  <input
    className="false input "
    disabled="disabled"
    name="name"
    onChange={[Function]}
    placeholder="Name"
    required={true}
    type="text"
    validations={Array []}
    value=""
  />
</div>
`;
